source/getting-started/faq.rst

```diff
@@ -351,6 +352,26 @@ ________________________________________________________________
 
 cc-snapshot is occasionally updated to accommodate changes to the infrastructure and distributions. To replace the script in your image, follow our instructions for :ref:`updating cc-snapshot <updating-snapshot>`.
 
+.. _move-images-between-regions:
+
+How to move images between regions?
+_____________________________________
+
+You can move images between regions by using the :ref:`command line interface <cli>`. Make sure you have :ref:`installed CLI properly <cli-installing>` and :ref:`configured the environment variables using the rc script <cli-rc-script>`.
+
+#. Download the image from the source region to local
+
+   .. code-block:: shell
+   
+       openstack image save <image_name> --file <filename> --os-region-name <source region_name [CHI@TACC or CHI@UC]>
```

Comment:
Please move the `--os-region-name` before `image save`, as it is a global optional argument.